src/main/Application.h

```diff
@@ -259,7 +259,7 @@ class Application
     // If config.ARTIFICIALLY_GENERATE_LOAD_FOR_TESTING=true, generate some load
     // against the current application.
     virtual void generateLoad(bool isCreate, uint32_t nAccounts,
-                              uint32_t offset, uint32_t nTxs, uint32_t txRate,
+                              uint32_t offset, uint32_t nOps, uint32_t opRate,
```

Comment:
on the ops vs txs front though: thinking about this more, I think we should keep reasoning based on txs. The core function of LoadGenerator (as least the way I understand it) is to stress test overlay/scp, not transaction application. So we should be able to deterministically manipulate what is flooded on the network (which we can do with txrate, but it's ambiguous with oprate). What do you think @hidenori-shinohara @MonsieurNicolas @graydon?

Reply:
I don't have a strong feeling, could go either way. Most of the time when people ask us about the performance of the network they're interested in ops-per-second, and ops-per-second is the meaningful unit for benchmarking anything that touches the database, which we have in the past used loadgen for, and may again in the future.

I agree that tx-per-second is the more meaningful unit for benchmarking rates of signature verification or txs-as-opaque-SCP-payloads. Given that the average op-per-tx level is probably close to 1 anyways, and we'll have metrics indicating the counts/rates of both in any given run, I'm not sure it is going to matter a lot.

Reply:
I'm good with either way. If either of them makes sense, I'd vote for keeping it as is for simplicity.

If we replaced it, any script containing the http endpoint parameters would have to be updated.

If one wants to specify the number of ops in loadgen, they can calculate the expected number of operations / tx based on the op-count distribution and use that to calculate `txrate` to achieve the desired `oprate`.

Reply:
Also, I just realized that keeping it as transactions would clarify some ambiguity around the implementation of this multi-op transaction feature.

If we passed `oprate` and the op-count distribution, the `oprate` and the op-count chosen from the distribution may be inconsistent. (e.g., `oprate` may be 5 ops / second, but we may pick 100 ops/tx for a certain transaction.)

However, if we passed `txrate` and the op-count distribution, we wouldn't have that issue since we would decide the number of transactions to submit and the number of operations in each transaction independently.

Reply:
Updated the logic based on `txrate` and `txs` rather than `oprate` and `ops`